src/app/page.tsx

```diff
@@ -33,14 +37,47 @@ export default function Home() {
     meetupUrl: 'https://www.meetup.com/dallas-software-developers-meetup/',
     communityUrl: '/',
     cohortUrl: '/',
+    githubUrl: 'https://github.com/dallassoftwaredevelopers',
+    discordUrl: '/',
+    linkedinUrl: '/',
```

Comment:
I'm unsure of the links for Discord and LinkedIn but can update once confirmed.

Reply:
Linked in: https://www.linkedin.com/company/dallas-software-developers
GitHub: https://github.com/dallassoftwaredevelopers
Meetup: https://www.meetup.com/dallas-software-developers-meetup/

Let me get back to you for the discord